rule_test.go

```diff
@@ -1038,6 +1038,30 @@ func TestRE(t *testing.T) {
 	}
 }
 
+func TestLastContent(t *testing.T) {
+	for _, tt := range []struct {
+		rule string
+		want string
```

Comment:
This test should probably not be dependent on the stringer implementation and should include the ContentOptions that we'd expect as well.

```
  for ... []struct {
    rule string
    want *Content
  }{
  ...
    rule = `...`
    want = &Content{...} 
  }

Reply:
Ok with `reflect.DeepEqual`